lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -49,6 +51,26 @@ def refresh
         end
       end
 
+      def timeout
+        logger.debug('job timed out')
+        super
+      end
+
+      def timeout_interval
+        execution_timeout_interval
+      end
+
+      def kill
+        ::Process.kill('SIGTERM', @command_pid)
+        publish_exit_status(2)
+        hosts = @inventory['all']['hosts']
+        hosts.each do |hostname|
+          @exit_statuses[hostname] = 2
+          publish_data_for(hostname, 'Timeout for execution passed, stopping the job', 'stdout')
```

Comment:
Would this message work as broadcasted by [`broadcast_data`](https://github.com/theforeman/smart_proxy_dynflow/blob/6b438eb154caeb211a557392f1ed5e3a8b2bcbaa/lib/smart_proxy_dynflow/runner/parent.rb#L36)?

Reply:
Yes definitely, why is it better to use the broadcast_data?